setup.py

```diff
@@ -202,6 +180,9 @@ def readme():
 
 str_libs = np_compile_args['libraries']
 
+if not is_win:
```

Comment:
Is this really needed?

Reply:
1. It is definitely not needed for building documentation. This line (and a lot of similar lines) were inherited from existing setup.py
2. After reviewing all platform specifics (is_win flag) I can conclude that it comes from the need to link third party dependencies that I would like to get rid of by the release:
* MPI 
* Boost (filesystem, system, regex)
* OpenCV

@Vyacheslav-Smirnov, comments?

Reply:
Yes, MPI and Boost are used
Opencv looks like is linked if installed on the system, I do not know if it is really used in SDC
In the conda recipe there is not dependency on opencv

Reply:
We do not need to carry on these dependencies. No MPI, no Boost, no OpenCV.